packages/frontend/src/components/transactions/TransactionsListItem.js

```diff
@@ -0,0 +1,39 @@
+import {Link} from "react-router-dom";
+import {getTransitionTypeString} from '../../util/index'
+import './TransactionsListItem.scss'
+
+
+function TransactionsListItem({transaction}) {
+    const hash = typeof transaction === 'object' ? transaction.hash : transaction;
+    const timestamp = transaction.timestamp;
+    const type = transaction.type;
+
+    return (
+        <Link 
+            to={`/transaction/${hash}`}
+            className='TransactionsListItem'
+        >
+
+            {typeof timestamp === 'string' &&
```

Comment:
I think theres a little sense in those condition, because all these fields will always be in the transaction object